### ISSUE TITLE
Not a valid aggregate or forensic report after fresh Install with 8.3.x

### ISSUE THREAD

**Issue by leonardo0014** [domainaware/parsedmarc#338]
Based on the information in ticket  #287 (Files https://github.com/domainaware/parsedmarc/issues/287#issuecomment-1029983415)  , I reinstalled the script on my SuSE Linux Version 15 in a virtual environment.
Unfortunately this didn't help, the error keeps coming

```
(parsedmarc) HOSTNAME:~ # parsedmarc -c /opt/ars/parsedmarc/etc/parsedmarc.ini --debug --verbose -o output ~/ALKANTPAN.CO.ZA\!firma.org\!1581318043\!1582020439\!3238.xml.gz

    INFO:cli.py:755:Starting parsedmarc

  0%|      | 0/1 [00:00<?, ?it/s]   DEBUG:__init__.py:935:Parsing /root/ALKANTPAN.CO.ZA!firma.org!1581318043!1582020439!3238.xml.gz

  0%|      | 0/1 [00:00<?, ?it/s]   ERROR:cli.py:837:Failed to parse /root/ALKANTPAN.CO.ZA!firma.org!1581318043!1582020439!3238.xml.gz - Not a valid aggregate or forensic report
```
The following versions were used:

- Package                  Version
- Python                  3.9
- aiohttp                  3.8.1
- aiosignal                1.2.0
- async-timeout            4.0.2
- attrs                    22.1.0
- azure-core               1.25.0
- azure-identity           1.11.0b2
- boto3                    1.24.46
- botocore                 1.27.46
- cachetools               5.2.0
- certifi                  2022.6.15
- cffi                     1.15.1
- charset-normalizer       2.1.0
- cryptography             37.0.4
- dateparser               1.1.1
- dnspython                2.2.1
- elasticsearch            7.13.4
- elasticsearch-dsl        7.4.0
- expiringdict             1.2.2
- frozenlist               1.3.1
- geoip2                   4.6.0
- google-api-core          2.8.2
- google-api-python-client 2.55.0
- google-auth              2.10.0
- google-auth-httplib2     0.1.0
- google-auth-oauthlib     0.5.2
- googleapis-common-protos 1.56.4
- html2text                2020.1.16
- httplib2                 0.20.4
- idna                     3.3
- IMAPClient               2.3.1
- jmespath                 1.0.1
- kafka-python             2.0.2
- lxml                     4.9.1
- mail-parser              3.15.0
- mailsuite                1.9.4
- maxminddb                2.2.0
- msal                     1.18.0
- msal-extensions          1.0.0
- msgraph-core             0.2.2
- multidict                6.0.2
- oauthlib                 3.2.0
- parsedmarc               8.3.0
- pip                      22.0.4
- portalocker              2.5.1
- protobuf                 4.21.4
- publicsuffix2            2.20191221
- pyasn1                   0.4.8
- pyasn1-modules           0.2.8
- pycparser                2.21
- PyJWT                    2.4.0
- pyparsing                3.0.9
- python-dateutil          2.8.2
- pytz                     2022.1
- pytz-deprecation-shim    0.1.0.post0
- regex                    2022.3.2
- requests                 2.28.1
- requests-oauthlib        1.3.1
- rsa                      4.9
- s3transfer               0.6.0
- setuptools               58.1.0
- simplejson              3.17.6
- six                      1.16.0
- tqdm                     4.64.0
- typing_extensions        4.3.0
- tzdata                   2022.1
- tzlocal                  4.2
- uritemplate              4.1.1
- urllib3                  1.26.11
- xmltodict                0.13.0
- yarl                     1.8.1

By the way, I've tested every version from 7.0.1 to the current one and got the same error.
Any idea what I'm doing wrong?

**Comment by leonardo0014** [comment]
Hi

Unfortunately, the latest update to 8.3.1 still didn't solve the problem. In order not to post a new ticket, I changed the title.

```
(parsedmarc) HOSTNAME:~ # parsedmarc -c /opt/ars/parsedmarc/etc/parsedmarc.ini /opt/ars/files/ALKANTPAN.CO.ZA.firma.org.1581318043.1582020439.3238.xml.gz --verbose --debug
    INFO:cli.py:777:Starting parsedmarc
  0%|      | 0/1 [00:00<?, ?it/s]   DEBUG:__init__.py:930:Parsing /opt/ars/files/ALKANTPAN.CO.ZA.firma.org.1581318043.1582020439.3238.xml.gz
  0%|      | 0/1 [00:02<?, ?it/s]
   ERROR:cli.py:859:Failed to parse /opt/ars/files/ALKANTPAN.CO.ZA.firma.org.1581318043.1582020439.3238.xml.gz - Not a valid aggregate or forensic report

```
is it possibly due to python 3.9?

**Comment by leonardo0014** [comment]
Hi

I finally figured out what my mistakes were.
Error 1 is in the example ini file. There at the parameter "ip_db_path". I only had the path "/usr/share/GeoIP" there, but not only the path but the mmdb file must be specified directly.
From the manual:
> ip_db_path - str: An optional custom path to a MMDB file from MaxMind or DBIP

This was probably changed or corrected after an update.

Now the script runs, but still doesn't output anything.
Error 2 was also in the sample ini file. There at the "silent" parameter, which was not set.

From the manual:
> silent - bool: Only print errors (Default: True)

In summary the solution in the .ini:
```
[general]
#ip_db_path = /usr/share/GeoIP/
silent = False
```

**Suggestion to the developers:**
a) Renames the "ip_db_path" parameter to "ip_db_file".
b) Outputs the error with the database not found on the CLI
